### PR TITLE
Bugfix/dle 1004 regnet recipes update

### PR DESCRIPTION
**Pull request by shaydeci**

### CODE REVIEW

File: src/super_gradients/recipes/imagenet_regnetY.yaml [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
+#  This example trains with batch_size = 256
+#  Training time on a single GeForce RTX 2080 Ti, and top1 accuracies:
+#  11 days for RegnetY200, 70.88
+#  12 days for RegnetY400, 74.74
+#  19 days for RegnetY600, 76.18
+#  20 days for RegnetY800, 77.07
+#  NOTE: Training should probably be lower as resources were shared among the above runs.
+
+#  Logs and tensorboards at:
+# https://deci-pretrained-models.s3.amazonaws.com/RegnetY800/
```
Comment (ofrimasad) [inline src/super_gradients/recipes/imagenet_regnetY.yaml:11]: changed to http format. was that intentional?

File: src/super_gradients/recipes/imagenet_regnetY.yaml [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
-#  Reach => 76.05 (Final averaged model)
-#  Log and tensorboard at s3://deci-pretrained-models/regnetY800-imagenet-tensorboard/
+#  RegnetY Imagenet classification training:
+#  This example trains with batch_size = 256
+#  Training time on a single GeForce RTX 2080 Ti, and top1 accuracies:
+#  11 days for RegnetY200, 70.88
+#  12 days for RegnetY400, 74.74
+#  19 days for RegnetY600, 76.18
+#  20 days for RegnetY800, 77.07
+#  NOTE: Training should probably be lower as resources were shared among the above runs.
```
Comment (ofrimasad) [inline src/super_gradients/recipes/imagenet_regnetY.yaml:8]: I think most users would like to parallel train this. You might want to add a comment stating is this is possible or not and how.

File: src/super_gradients/recipes/imagenet_regnetY.yaml [hunk 3]
```diff
@@ -38,7 +50,7 @@ data_loader_num_workers: 8
 model_checkpoints_location: local
 load_checkpoint: False
 
-experiment_name: regnetY800_imagenet
+experiment_name: regnetY800_imagenet_updated
```
Comment (ofrimasad) [inline src/super_gradients/recipes/imagenet_regnetY.yaml:53]: intentional?

**Review by ofrimasad — COMMENTED** [review]
my comments inline

File: src/super_gradients/recipes/imagenet_regnetY.yaml [hunk 4]
```diff
@@ -1,14 +1,26 @@
...
+#  Logs and tensorboards at:
+# https://deci-pretrained-models.s3.amazonaws.com/RegnetY800/
+# https://deci-pretrained-models.s3.amazonaws.com/RegnetY600/
+# https://deci-pretrained-models.s3.amazonaws.com/RegnetY400/
+# https://deci-pretrained-models.s3.amazonaws.com/RegnetY200/
 
 # Instructions:
 # Set the PYTHONPATH environment variable: (Replace "YOUR_LOCAL_PATH" with the path to the downloaded repo):
-#   export PYTHONPATH="YOUR_LOCAL_PATH"/super_gradients/
+#   export PYTHONPATH="YOUR_LOCAL_PATH"/super_gradients/:"YOUR_LOCAL_PATH"/super_gradients/src/
```
Comment (ofrimasad) [inline src/super_gradients/recipes/imagenet_regnetY.yaml:18]: now that you moved recipes into src you dont need `"YOUR_LOCAL_PATH"/super_gradients/` anymore

File: src/super_gradients/recipes/imagenet_regnetY.yaml (hunk 2 above)
Comment (shaydeci) [reply src/super_gradients/recipes/imagenet_regnetY.yaml:8]: We haven't tested that if we parallelise we reproduce these results...so I think it's a bad idea to add such comment. If a user wants to try and reproduce theses results while using DDP, I think it should be at their own responsibility.

**Review by ofrimasad — COMMENTED** [review]
just the experiment name. 
regnetY800_imagenet_updated?  whats next "regnetY800_imagenet_updated_new"?

**Pull request merged**